### ISSUE TITLE
Design horizontal masonry grid slider with slick 

### ISSUE THREAD

**Issue by asmsaiff** [kenwheeler/slick#4274]
How can I design this slider in slick slider ?

https://www.figma.com/proto/11soIQr2TkHMYyEgMJY1aW/Untitled?type=design&node-id=401-69&t=BXwESy12VKdSmXLx-0&scaling=scale-down-width&page-id=0%3A1

**Comment by markvantilburg** [comment]
@ahmadalfy please close this issue, there is nothing to see

**Comment by asmsaiff** [comment]
> [@ahmadalfy](https://github.com/ahmadalfy) please close this issue, there is nothing to see

No reply for the last 3 years, and you say there is nothing to see.  

Funny man. 😂😂😂

**Comment by ahmadalfy** [comment]
We're trying to clear up old issues and we will be glad to assist if this is a still a problem you need help with.

**Comment by asmsaiff** [comment]
No, thank you. Who are you waiting for help from to resolve the issue in this AI era?

**Comment by ahmadalfy** [comment]
Well, if it's able to help you that's great. We'll be glad to help if the AI falls short. Have a nice day
